sample-cap/srv/foundation-models/azure-openai-service.ts

```diff
@@ -1,8 +1,7 @@
-import { Request } from '@sap/cds';
 import { AzureOpenAiChatClient } from '@sap-ai-sdk/foundation-models';
 
 export default class AzureOpenAiService {
-  async chatCompletion(req: Request) {
+  async chatCompletion(req: any) {
```

Comment:
why removing the type and using any? that doesnt seem correct.

Please import cds and use it like this

```typescript
import cds from "@sap/cds"
...
async chatCompletion(req: cds.Request) {
...
```

edit: this applies to all usages of `any`

Reply:
As explained, I face the following error:

```
Could not find a declaration file for module '@sap/cds'. '****/ai-sdk-js/node_modules/.pnpm/@sap+cds@8.4.1_express@4.21.1/node_modules/@sap/cds/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sap__cds` if it exists or add a new declaration (.d.ts) file containing `declare module '@sap/cds';`ts(7016)
```

You see that I already include `@cap-js/types` in `package.json` and I tried all ways to installed it also with `@sap/cds`. But it won't work.

Reply:
Discussed. And check known issue.